src/shaders/Sky.tsx

```diff
@@ -1,11 +1,16 @@
 import React, { forwardRef, useMemo } from 'react'
-import { ReactThreeFiber, extend } from 'react-three-fiber'
+import { ReactThreeFiber } from 'react-three-fiber'
 import { Sky as SkyImpl } from 'three/examples/jsm/objects/Sky'
 import { Vector3 } from 'three'
 
 export type Sky = {
```

Comment:
Wait isn't this entirely useless?

Reply:
I mean, defining the component props and JSX declaration, we are using a `<primitive>` after all

Reply:
@gsimone am I insane? lol

Reply:
I don't think I'm at all qualified to answer that haha, I've just started playing around with THREE and r3f in the last week. What specifically are you referring to by "this"? The fact that I removed extend? Or importing `ReactThreeFiber`at all?